video/external/src/decoder.rs

```diff
@@ -0,0 +1,11 @@
+// bindgen ../openh264/codec/api/wels/codec_api.h --no-prepend-enum-name \
```

Comment:
Could this be done through a `build.rs` script, instead of committing the file?

Reply:
From a technical PoV, of course.
Then, we'd have to vendor (or perhaps obtain in some other way) the C header in question at build time.
I think the license within it allows vendoring, and the header in itself should not be problematic regarding patents or royalties.
That is a smaller file to check in, at the price of another dependency and a smidge longer build time.
Would you prefer that?

Reply:
Could you please weigh in on this one too, @Dinnerbone?

Reply:
I don't care enough to block this PR. If someone feels strongly about it, let's follow up in another PR later.